src/main/java/seedu/multitasky/logic/parser/ListCommandParser.java

```diff
@@ -25,28 +29,81 @@ public ListCommand parse(String args) throws ParseException {
         if (trimmedArgs.isEmpty()) {
             return new ListCommand();
         }
-
-        // Flags are delimited by whitespace
-        final String[] flags = trimmedArgs.split("\\s+");
-        if (!hasValidFlagCombination(flags)) {
+        // Preamble is necessary due to how ArgumentTokenizer works
```

Comment:
and the reason for looking for _Prefix_ is to prevent unnecessary matching like `by`stander or `on`i`on`
the issue could probably be fixed by making tokenizer add a white space before and after trimmedArgs internally and then adjust the numbers that it works on